website/docs/presets/gql-tag-operations.md

```diff
@@ -0,0 +1,102 @@
+---
+id: gql-tag-operations
+title: gql-tag-operations
+---
+
+This preset generates typings for your inline `gql` function usages, without having to manually specify import statements for the documents. All you need to do is import your `gql` function and run codegen in watch mode.
+
+Huge thanks to [Maël Nison](https://github.com/arcanis), who conceptualized the foundation for this preset [over here](https://github.com/arcanis/graphql-typescript-integration).
```

Comment:
This is nice and all but the MIT license requires you to include a full copy of it in the source code

Reply:
can u send a pr?